doc/specs/Font features and axes-spec.md

```diff
@@ -0,0 +1,92 @@
+
+We will do this by allowing users to define a dictionary in their settings.json file, where the keys are the 4-character feature tags and the values are the parameter values. This dictionary will then get applied to our internal dictionary (which will contain the standard list of 11 features with their parameter values), meaning that any new key-value pairs will get added to our dictionary and any existing key-value pairs will get updated. Finally, this 'merged' dictionary will be what we use to construct the list of features to pass into DWrite.
+
+### **Axes of variation**
+
+Specifying axes of variation is done in an extremely similar manner to the way font features are specified - a 4-character tag is used to specify which font axis is being modified and a numerical value is provided to specify the value the axis should be set to. For example, {'slnt', 20} specifies that the 'slant' axis should be set to 20.
+
+There is also a standard list of axes of variation, and each axis has its own default. We will approach this the same way we approached font features, by allowing users to specify additional features or omit features without needing to redefine the defaults.
+
+## UI/UX Design
+
+Users will be able to add a new setting to their font objects (added in [#10433](https://github.com/microsoft/terminal/pull/10433)). The resultant font object may look something like this
+
+```
+"font": {
+    "face": "Cascadia Code",
+    "size": 12,
+    "features": {
+        "ss03": 1,
+        "liga": 0
+    },
+    "axes": {
+        "slnt": 20.5
+    }
+}
+```
+
+## Capabilities
+
+### Accessibility
+
+Should not affect accessibility.
+
+### Security
+
+Should not affect security.
+
+### Reliability
+
+Aside from additional parsing required for the settings file (which inherently offers more locations for parsing to fail), we need to be careful about badly formed/non-existant feature tags or axes specified in the user-defined dictionaries. We must make sure to ignore such declarations (perhaps alongside emitting a warning to the user) and only apply those that are correctly formed and exist.
+
+### Compatibility
+
+Older versions of Windows may not have the DWrite updates that allow for defining font features and axes of variation. We must make sure to fallback to the current implementation in these cases.
+
+### Performance, Power, and Efficiency
+

```

Comment:
```suggestion
### Performance, Power, and Efficiency

Should not affect performance, power, or efficiency.

```
presumably, this is what you meant to do

Reply:
Thanks for catching that!